File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// MixpanelLexiconSchemas SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class MixpanelLexiconSchemasMakeContext
{
    public static function call(array $ctxmap, ?MixpanelLexiconSchemasContext $basectx): MixpanelLexiconSchemasContext
    {
        return new MixpanelLexiconSchemasContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/py/pkg/utility/make_context.py
# MixpanelLexiconSchemas SDK utility: make_context

from projectname_sdk.core.context import MixpanelLexiconSchemasContext


def make_context_util(ctxmap, basectx):
    return MixpanelLexiconSchemasContext(ctxmap, basectx)
